be/src/service/brpc_service.h

```diff
@@ -18,10 +18,12 @@
 #pragma once
 
 #include <memory>
-
+#include "service/brpc.h"
```

Comment:
You'd better not include this file in `.h` file, because it will make other files which include this file will include `service/brpc.h`.